Automattic/themes
# Blockbase: revise padding

This PR uses CSS classes to add padding around the header, post titles + featured image, and post content.

I thought this was a better approach than adding the styles inline via block attributes, because it matches how we are adding the padding around the post content.

**To test**
- Add some posts that have a featured image
- Check the index / query and singular views 
- Ensure the padding is correct (20px unless aligned full)

Before | After
------- | --------
<img width="450" alt="Screen Shot 2021-05-27 at 12 43 09 PM" src="https://user-images.githubusercontent.com/5375500/119865163-3ebd4b00-bed0-11eb-8a54-55c9612dd264.png"> | <img width="463" alt="Screen Shot 2021-05-27 at 12 43 36 PM" src="https://user-images.githubusercontent.com/5375500/119865217-4b41a380-bed0-11eb-9d7f-7a0172643ff0.png">

Closes #3963.
